Packs/Yara/Playbooks/playbook-YARA_-_File_Scan.yml

```diff
@@ -268,10 +268,10 @@ tasks:
           root: inputs.EntryID
           transformers:
           - operator: uniq
-      yaraRule:
+      yaraRules:
```

Comment:
Please note that changing this value, and not changing the YML - breaking the original logic.
Please revert those changes in PB.
Also, let's keep the original script argument name - so it won't break any logic for other users.